includes/frontend/class-wcmp-frontend.php

```diff
@@ -77,13 +77,25 @@ public function confirmationOrderReceived(int $order_id): void
      */
     private function isLocalPickup(): bool
     {
-        $shippingMethodString = WC()->session->get('chosen_shipping_methods')[0] ?? '';
+        $shippingMethodString = $this->getShippingMethodString();
 
         [$methodSlug] = WCMP_Checkout::splitShippingMethodString($shippingMethodString);
 
         return WCMP_Shipping_Methods::LOCAL_PICKUP === $methodSlug;
     }
 
+    /**
+     * @return string
+     */
+    private function getShippingMethodString(): string
+    {
+        if(! isset(WC()->session)) {
+            return '';
```

Comment:
Liever geen lege strings gebruiken, dan is het beter om null te returnen en expliciet in bovenstaande functie te controleren of er wat uit is gekomen.